src/component.js

```diff
@@ -1,59 +1,76 @@
 /* @flow */
 
-import { getClientID, getMerchantID, getPayPalDomain, getVersion, isPayPalDomain, getEventEmitter, getDebug, getEnv, getVault, getSDKQueryParam } from '@paypal/sdk-client/src';
-import { UNKNOWN, ENV } from '@paypal/sdk-constants/src';
-
-import { logger } from './lib/logger';
-
-export const PPTM_ID = 'xo-pptm';
+import {
+  getClientID,
+  getMerchantID,
+  getPayPalDomain,
+  getVersion,
+  isPayPalDomain,
+  getEventEmitter,
+  getDebug,
+  getEnv,
+  getVault,
+  getSDKQueryParam,
+} from "@paypal/sdk-client/src";
+import { UNKNOWN, ENV } from "@paypal/sdk-constants/src";
+
+import { logger } from "./lib/logger";
+
+import { waitForWindowReady } from "belter/dist/module";
```

Comment:
Are we able to import this from `belter/src` instead?
```suggestion
import { waitForWindowReady } from "belter/src";
```